src/components/DateRange/DateRange.js

```diff
@@ -0,0 +1,73 @@
+import React, { useRef, useEffect, useState } from "react";
+
+import DateTimePicker from "../DateTimePicker";
+
+const DateRange = props => {
+  const {
```

Comment:
Can we specify date vs time vs datetime?

Reply:
Can we have a min max?

Reply:
yes, we can, i put some demo cases on HomeScreen (click login button for fake authorization)